libraries/stdlib/genglsl/stdlib_genglsl_impl.mtlx

```diff
@@ -561,7 +561,7 @@
   <implementation name="IM_screen_color4_genglsl" nodedef="ND_screen_color4" target="genglsl" sourcecode="({{mix}}*((1.0 - (1.0 - {{fg}}) * (1.0 - {{bg}})))) + ((1.0-{{mix}})*{{bg}})" />
 
   <!-- <overlay> -->
-  <implementation name="IM_overlay_float_genglsl" nodedef="ND_overlay_float" target="genglsl" sourcecode="({{fg}} < 0.5) ? ({{mix}}*2.0*{{fg}}*{{bg}}) + ((1.0-{{mix}})*{{bg}}) : ({{mix}}*(1.0-(1.0-{{fg}})*(1.0-{{bg}}))) + ((1.0-{{mix}})*{{bg}})" />
+  <implementation name="IM_overlay_float_genglsl" nodedef="ND_overlay_float" target="genglsl" sourcecode="({{bg}} < 0.5) ? ({{mix}}*2.0*{{fg}}*{{bg}}) + ((1.0-{{mix}})*{{bg}}) : ({{mix}}*(1.0-2.0*(1.0*{{fg}})*(1.0-{{bg}}))) + ((1.0-{{mix}})*{{bg}})" />
```

Comment:
I think `({{mix}}*(1.0-2.0*(1.0*{{fg}})*(1.0-{{bg}})))` should be 
`({{mix}}*(1.0-2.0*(1.0-{{fg}})*(1.0-{{bg}})))` instead.

Reply:
Fixed

Reply:
👍